.github/actions/nodejs/action.yaml

```diff
@@ -17,33 +17,16 @@ runs:
       shell: bash
       run: npm config set audit false
 
-    - name: Configure AWS credentials and bucket region
-      uses: aws-actions/configure-aws-credentials@v4
-      if: contains(runner.name, 'ubuntu-platform')
-      with:
-        aws-region: eu-west-1
-
     - name: Cache NPM build artifacts (S3 bucket cache)
-      uses: everpcpc/actions-cache@v1
-      if: contains(runner.name, 'ubuntu-platform')
+      uses: strophy/actions-cache@opendal-update
```

Comment:
We should prefer `@master`, + we should migrate that repo to dashpay org

Reply:
We expect some updates on upstream soon. When they don't we will change it.